index.js

```diff
@@ -58,6 +58,31 @@ function onInitHooks() {
 	)
 }
 
+function onInitCreateAdmin() {
+	const email = process.env.BALENA_ROOT_EMAIL || '';
```

Comment:
These should be coming from `config.ts` which handles grabbing env vars/defaulting if necessary

Reply:
done